.github/workflows/promote-config.yml

```diff
@@ -26,8 +26,16 @@ jobs:
             src_stream=testing-devel
           elif [ "${target_stream}" == next ]; then
             # promote from testing-devel while we are in lockstep
-            # https://github.com/coreos/fedora-coreos-pipeline/pull/425
-            src_stream=testing-devel
+            enabled="$(curl https://raw.githubusercontent.com/coreos/fedora-coreos-pipeline/main/next-devel/status.json | jq .enabled)"
```

Comment:
`-o pipefail` isn't turned on here (unless `run` implies that?) and `jq` won't error out if it's passed an empty body. Maybe let's split this into:

```sh
curl -LO ...
enabled=$(jq .enabled status.json)
```

and also for good measure add `set -euo pipefail` at the top?

There's also `jq -e` but that won't allow us to differentiate the null/missing key case.

Reply:
The [default shell](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsshell) does set `pipefail`, but there's a fallback to `sh` which doesn't, so I think it makes sense to be explicit.  Having done that, I think it's not necessary to also split up the pipeline.